src/templates/docs/measure/index.html

```diff
@@ -67,8 +70,9 @@ <h1 class="f5 fw4 sans-serif pt5 bt b--black-10 ttu black"> <b>3rem(48px)</b> at
             no sea takimata sanctus est Lorem ipsum dolor sit amet.</span>
           </p>
         </div>
-        <h1 class="f6 pt5 mb4 bt b--black-10 ttu black"><b>2.25rem (36px)</b> at <b>30em</b></h1>
-        <div class="cf w-100 pb4">
+        <h1 class="f6 pt5 bt b--black-10 ttu black"><b>2.25rem (36px)</b> at <b>30em</b></h1>
```

Comment:
Removed `mb4` and instead applied to example below as margin top, this is so the class name usage example were evenly spaced